tests/unit/backend/corpora/common/test_authorizer.py

```diff
@@ -1,23 +1,27 @@
+import json
 import os
 import unittest
 
 import requests
 from chalice import UnauthorizedError
 
 from backend.corpora.common.authorizer import assert_authorized
+from backend.corpora.common.utils.aws_secret import AwsSecret
 
 if not os.getenv("DEPLOYMENT_STAGE"):  # noqa
```

Comment:
More of a curiosity question - why does the deployment stage need to be `test` instead of `dev`?

Reply:
should be dev